youtube-client-app/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  Component, EventEmitter, OnInit, Output,
+} from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss'],
+})
+export class HeaderComponent implements OnInit {
+  @Output() public toggleSetting: EventEmitter<void> = new EventEmitter();
```

Comment:
you don't have to specify a type - you can just pass generic to EventEmitter's constructor like this:
`@Output() public toggleSetting = new EventEmitter<void>();`

Reply:
fixed